core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -316,8 +316,14 @@ class ConsumerBounceTest extends AbstractConsumerTest with Logging {
     // roll all brokers with a lesser max group size to make sure coordinator has the new config
     val newConfigs = generateKafkaConfigs(maxGroupSize.toString)
     var kickedOutConsumerIdx: Option[Int] = None
+    val holdingGroupBrokers = servers.filter(!_.groupCoordinator.groupManager.currentGroups.isEmpty).map(_.config.brokerId)
+    // should only have one broker holding the group metadata
+    assertEquals(holdingGroupBrokers.size, 1)
+    val coordinator = holdingGroupBrokers.head
+    // ensure the coordinator broker will be restarted first
+    val orderedBrokersIds = List(coordinator) ++ servers.indices.toBuffer.filter(_ != coordinator)
```

Comment:
I'm a little confused about the problem we're fixing. Why does it matter that the coordinator is restarted first? Overall, I wonder if we can simplify this. Our goal is to observe the effect of changing the max group size config. Could we restart all brokers first and then check the consumers for the exception we expect?